course/templates/course/staff/_tag_remove_modal.html

```diff
@@ -18,32 +18,28 @@
                     <span aria-hidden="true">&times;</span>
                 </button>
                 <h4 class="modal-title" id="tag-remove-modal-label">
-                    {% trans "Untag multiple users" %}
+                    {% translate "UNTAG_MULTIPLE_USERS" %}
                 </h4>
             </div>
             <div class="modal-body">
-                {% blocktrans %}
-                Remove tagging
-                <span id="tag-remove-modal-tagging"></span>
-                from <span id="tag-remove-modal-amount"></span> users:
-                <ul id="tag-remove-modal-users"></ul>
-                {% endblocktrans %}
+                {% blocktranslate trimmed %}
+                REMOVE_TAGGING_FROM_USERS_MODAL{% endblocktranslate %}
```

Comment:
This {% endblocktranslate %} tag should probably be on its own row, since the {% blocktranslate %} tag is on its own row.

Reply:
Good catch, thanks, I'll fix this!